tests/test_rpc.py

```diff
@@ -45,6 +51,19 @@ def assert_attr(self, **kwargs):
         __tracebackhide__ = True
         return assert_attr(self.__node, **kwargs)
 
+    def assert_property_present(self, name):
```

Comment:
This seems like too much syntactic sugar to me.

Reply:
Do you think it would be better if I removed the methods `assert_property_present` & `assert_property_absent` and just left `property_present`?

Reply:
That makes sense to me. Just use regular `assert` calls for those two operations.

Reply:
+1 to using just `propery_present`.